### DISCUSSION TITLE
Building fabric api for 1.14 causes mixin injection error

### DISCUSSION THREAD

**Discussion by u9g** [FabricMC/fabric-api#2209] [Mod Dev Support] [answered]
Error: https://gist.github.com/u9g/a2225549061b2868f2072ba582dd1806

**Comment by modmuss50** [comment, marked as answer]
Your using fabric `0.28.5+1.14` this was released for 1.14.4 and not 1.14. See the [curseforge ](https://www.curseforge.com/minecraft/mc-mods/fabric-api/files/all?filter-status=1&filter-game-version=2020709689%3A7469) page for all the versions. You may need to use an older version that is known to work on 1.14 or update to 1.14.4.

**Reply by u9g** [reply]
Great, thanks!
